Runtime/BackgroundDownload.cs

```diff
@@ -110,7 +114,10 @@ public enum BackgroundDownloadStatus
     /// A class to perform file downloads that will continue if app goes into background or even gets shut down by OS.
     /// Note, that the download might not finish under specific conditions, for example Operating System can provide user with a feature to cancel such download.
     /// An instance of this class can be returned from the Coroutine to suspend it until download is finished.
-    /// The object of this class must be disposed when no longer required by calling Dispose().
+    /// The object of this class must be disposed when no longer required by calling Dispose() or by placing the object in the using() block.
+    /// If background download is disposed before completion, it is also cancelled, the existence and contents of result file is undefined.
```

Comment:
Clarifying question: What do we mean by "the existence and contents of result file is undefined"?

Reply:
It means the result file may or may not exist, contain old data or contain partial data.